src/visitors/semantic/completion/mod.rs

```diff
@@ -192,15 +192,19 @@ impl<'a> Visitor<'a> for CompletableObjectFinderVisitor {
 }
 
 fn get_var_type(expr: &Expression) -> Option<VarType> {
+    match expr.type_of() {
```

Comment:
also curious, why was this needed here?

Reply:
in the case of something like `-1h` this was actually showing up as a unary expression rather than a duration but if we use the type inference from flux we can determine the type.

Reply:
ahh nice find, the remaining types in the match below this, are we not able to use the `type_of()` for them? curious why there is a split 🤔

Reply:
they probably can but I want to do that in a different PR

Reply:
the remaining ones will be a larger change

Reply:
right on, thanks for sharing 👍